.github/workflows/bump.yml

```diff
@@ -56,19 +57,17 @@ jobs:
       - name: Bump version
         run: ${{ inputs.bump-command }}
 
-      - name: Get package.json version
-        run: echo VERSION=$(node -p 'require("./package.json").version') >> $GITHUB_ENV
-
       - name: Commit and push
         run: |
           git config --local user.email "deployooor@nexusmutual.io"
           git config --local user.name "Version bumpooor"
-          git add .
-          git commit -m "chore: release v${{ env.VERSION }} [skip ci]"
-          git push origin ${{ inputs.ref }}
 
-      - name: Tag
-        if: inputs.ref == 'master'
-        run: |
-          git tag v${{ env.VERSION }}
-          git push origin --tags
+          # check if there are changes to commit
+          if [[ -z "$(git status --porcelain package.json package-lock.json)" ]]; then
+            echo "No changes to commit. Failing bump job."
+            exit 1
+          fi
```

Comment:
fails the `bump.yml` workflow if there were no version bump from the bump command